src/main.rs

```diff
@@ -0,0 +1,42 @@
+use clap::{App, Arg};
+
+use lmdb_utils::{db, error::ToolError};
```

Comment:
```suggestion
use casper_db_utils::{db, error::ToolError};
```

Required for build without errors for me.